src/main/java/org/graylog2/gelfclient/transport/GelfSenderThread.java

```diff
@@ -38,23 +38,28 @@
  */
 public class GelfSenderThread {
     private static final Logger LOG = LoggerFactory.getLogger(GelfSenderThread.class);
+
     private final ReentrantLock lock;
     private final Condition connectedCond;
     private final AtomicBoolean keepRunning = new AtomicBoolean(true);
     private final Thread senderThread;
     private Channel channel;
     private final int maxInflightSends;
+    private BlockingQueue<GelfMessage> queue;
```

Comment:
The `queue` and `inflightSends` fields should be `final` so they cannot be set again by accident.